.github/workflows/continuous-integration.yml

```diff
@@ -10,8 +10,23 @@ jobs:
       - name: Build docker prod image
         run: make build-prod
 
-      - name: Build docker dev image
-        run: make build-dev
+      - name: Run prod image
+        run: make run-prod
 
-      - name: Run the tests
-        run: make run-tests
+      - name: Build docker test image
```

Comment:
not needed since it is run with `run-pre-test` automatically

Reply:
You're right. I added this as a separate step to have a better "logging", if a step fails.